octobot_trading/modes/channel/abstract_mode_producer.py

```diff
@@ -441,7 +441,10 @@ async def _register_and_apply_required_user_inputs(self, context):
                 common_enums.ActivationTopics.EVALUATION_CYCLE.value,
                 activation_topic_values
             )
-        await self._apply_exchange_side_config(context)
+        try:
+            await self._apply_exchange_side_config(context)
+        except Exception as err:
+            self.logger.exception(err, True, f"Error when applying exchange side config: {err}")
```

Comment:
Should we really continue if side config apply failed?

Reply:
For now this is only affecting the leverage setting, if the exchange leverage is different from the configured one. Do you think we should stop the trading mode instead ? (as a note, this error was not caught before and was hidden as this is run in a task)

Reply:
I don't know, I wanted to know your point of view about it. I'm not sure we should continue even if it's only the leverage setting, we are in an unhealthy state.